src/azfswatchdog

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash 
+
+#  
+# How often does the watchdog look for unmounts and/or IP address changes for 
+# Blob endpoints. 
+# 
+
+MONITOR_INTERVAL_SECS=5 
+
+# 
+# This stores the map of local IP and share name and external blob endpoint IP. 
+# 
+
```

Comment:
nit, don't leave this line between comment and the code

Reply:
done.